styles/utlis.css

```diff
@@ -0,0 +1,34 @@
+
+:root {
+  --primary-color: #6b46c1;
+  --secondary-color: #f9f9f9;
+  --shadow-color: #bbb1c0;
+}
+
+.ml-2 {
+  margin-left: 2rem;
+}
+.p1 {
+  padding: 0 1rem;
+}
+
+.btn {
+  border: none !important;
+}
+.form {
+  border: none !important;
+  padding: 0;
+}
+.logo {
+  color: var(--primary-color);
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
Maybe add utility classes for flex, flex-direction, justify-content, items, etc. Will help you in the future while developing the app and promote reusablity.